canonical-websites/www.ubuntu.com
# Update openstack consulting page

## Done

Update `openstack/consulting`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack/consulting>
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1f83OSIHVQGMjF7b0pZUKFvoubMGqi_b-OYbSpyhveX4/edit#)


## Issue / Card

Fixes [#637](https://github.com/ubuntudesign/web-squad/issues/637)
